client/src/components/employer-form/Review.js

```diff
@@ -74,32 +73,7 @@ export const Review = ({ handleEditClick, isDisabled }) => {
               <Grid container alignItems="center" justify="space-between" spacing={2}>
                 <Grid item>
                   <Typography variant="subtitle1">
-                    3. Site Type and Size
-                  </Typography>
-                </Grid>
-                {!isDisabled && (
-                  <Grid item>
-                    <Button
-                      startIcon={<EditIcon />}
-                      fullWidth={false}
-                      size="small"
-                      onClick={() => handleEditClick(3)}
```

Comment:
all of these values will need to be updated too - same thing I missed last night